cli/tests/run_tests.rs

```diff
@@ -2780,6 +2780,12 @@ mod run {
     exit_code: 1,
   });
 
+  itest!(error_with_errors_prop {
```

Comment:
Please add a comment here linking to relevant issue - it's not immediately obvious what we are testing here.